image/scripts/do_overwrite.sh

```diff
@@ -16,26 +15,17 @@ cp gnome-control-center gnome-control-center.real
 cp unity-control-center unity-control-center.real
 popd > /dev/null
 
-# Then, disable all X sessions except one
-# TODO: This does not seem to work very well - investigate!
-#pushd /usr/share/xsessions > /dev/null
-#mv gnome.desktop gnome.desktop.backup
-#mv gnome-shell.desktop gnome-shell.desktop.backup
-#mv ubuntu.desktop ubuntu.desktop.backup
-#mv ubuntu-2d.desktop ubuntu-2d.desktop.backup
-#popd > /dev/null
-
 # Now do the deed
 cp -r ../overwrites/* $DESTINATION
 
 # Permissions fixup
 chmod 0440 ${DESTINATION}etc/sudoers.d/keep-proxy
+
 # Remove Bluetooth indicator applet from Borger user
-BLUETOOTH_INDICATOR_PATH=$(find /usr/lib -name 'indicator-bluetooth-service')
-if [ ! -z "$BLUETOOTH_INDICATOR_PATH" ]
-then
-    chmod o-x $BLUETOOTH_INDICATOR_PATH
-fi
+DIR=$(dirname $(realpath $0 ))
+"$DIR/../../admin_scripts/image_core/remove_bluetooth_applet.sh"
```

Comment:
This is an _exceedingly_ subtle point, but this script doesn't seem to have the execute bit set and so isn't runnable:

```
$ ls -la remove_bluetooth_applet.sh 
-rw-r--r-- 1 alec alec 1106 Nov 19 11:01 remove_bluetooth_applet.sh
```